scenario.py

```diff
@@ -76,13 +77,23 @@ def __init__(self, params, experiment_path):
         else:
             self.epoch_count = 40
             
-        if 'methods' in params:
-            self.methods = params['methods']
+        methods_default = ["Shapley values",
+                           "Independant scores",
+                           "TMCS values"
+                           ]
```

Comment:
For readability, would you kindly format the list as following?
```python
list = [
    "one",
    "two",
    "three",
]

Reply:
Sure!